easybuild/tools/filetools.py

```diff
@@ -1162,6 +1163,50 @@ def rmtree2(path, n=3):
     else:
         _log.info("Path %s successfully removed." % path)
 
+def find_backup_name_candidate(src_file):
+    """Returns a non-existing file to be used as destination for rotating backups"""
+    dst_file = src_file
```

Comment:
Hmm, shouldn't the default be `None`?

If the specified filepath doesn't exist, you can't back up that file either?